pm_muairss_write/pm_muairss_write.xml

```diff
@@ -3,27 +3,46 @@
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
         pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
-        cp $structure input.cell &&
-        cp $params params.yaml &&
+        ln -s $structure input.cell &&
+        ln -s $params params.yaml &&
+        #if str($iscastep.iscastep_sel)=="true":
+            ln -s $castep_param castep.param &&
+        #end if
         pm-muairss -t w input.cell params.yaml &&
```

Comment:
If `params.yaml` does not have `castep_param` set, pm-muairss does not pick up the `castep.param` file at runtime (in fact, this is probably true if `castep_param` is set to anything other than `castep.param`). It therefore defaults to a generic CASTEP param file and sticks that in all the output folders instead. We need to make sure this parameter file actually gets passed through to pm-muairss (by adding it to params.yaml if it's not there, or tweaking the line if it is).

This also means that no error is thrown if calculator is castep but iscastep_sel is false – perhaps this shouldn't throw an error if it does fall back on defaults, but it should maybe raise a warning.

Reply:
fixed, now muairss write overwrites the castep param config value with the loaded file. Also updated castep defaults from pymuonsuite schema

Reply:
It should also add the castep_param line if it isn't already present, IMO.

What do you mean by the CASTEP defaults?

Reply:
https://github.com/muon-spectroscopy-computational-project/pymuon-suite/blob/master/pymuonsuite/schemas.py

Reply:
edited sed to add castep param if missing.